internal/k8shandler/metric_exporter.go

```diff
@@ -0,0 +1,61 @@
+package k8shandler
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/cluster-logging-operator/internal/metrics/telemetry"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+
+	log "github.com/ViaQ/logerr/v2/log/static"
+	loggingv1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	loggingv1alpha1 "github.com/openshift/cluster-logging-operator/apis/logging/v1alpha1"
+	logfilemetricexporter "github.com/openshift/cluster-logging-operator/internal/metrics/logfilemetricexporter"
+	"github.com/openshift/cluster-logging-operator/internal/utils/k8s_objects"
+	"k8s.io/client-go/tools/record"
+	client "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+)
+
+func ExporterMetric(lmfeInstance *loggingv1alpha1.LogFileMetricExporter,
```

Comment:
Moving to a LFME specific package I would Rename this to "Reconcile"

Reply:
done